regen-prelaunch-3/genesis.tmpl.json

```diff
@@ -1,5 +1,5 @@
 {
-  "genesis_time": "2021-04-12T19:00:00Z",
+  "genesis_time": "2021-04-12T19:50:00Z",
```

Comment:
Why? It doesn't matter if we start after the genesis time right?

Reply:
Yes, it doesn't. We were bit confused as it was giving some errors. But later found that it was  due to some other nodes connected to prelaunch-2 and they were producing blocks